pyproject.toml

```diff
@@ -17,6 +17,7 @@ dependencies = [
   "inspect_ai >= 0.3.118",
   "pydantic >= 2.11, <3",
   "pydantic-settings",
+  "datasets >= 4.0.0",
```

Comment:
```suggestion
```

this is not a dependency of our extension, only of most evals one might run - not our place to manage it for the user